src/Bootstrap.php

```diff
@@ -92,4 +94,10 @@ public static function removeNotFoundRedirects(Redirect $redirect)
         $notFoundRepository = new NotFoundRepository(new DB());
         $notFoundRepository->deleteByUrlAndLocale($redirect->getFrom(), $redirect->getLocale());
     }
+
+    public static function allowUserInSitemap(bool $allowInSitemap, int $userID)
+    {
+        $isPublic = get_user_meta($userID, 'public', true);
+        return $isPublic && $isPublic === '1';
```

Comment:
Boolval checker korrekt imod '0' og '1': https://www.php.net/manual/en/function.boolval.php
```suggestion
        $isPublic = get_user_meta($userID, 'public', true);
        return boolval($isPublic);
```